### PR TITLE
Add Machine Pools commands

### PR DESCRIPTION
**Pull request by igoihman**

### CODE REVIEW

**Comment by nimrodshn** [comment]
@igoihman I think you need to bump the sdk version to version `0.0.89` (https://github.com/openshift-online/ocm-sdk-go/pull/271/files)

File: cmd/ocm/list/machinepool/cmd.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	clusterKey string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "machinepools",
+	Aliases: []string{"machine-pool", "machinepools", "machinepool"},
```
Comment (vkareh) [inline cmd/ocm/list/machinepool/cmd.go:38]: `machinepools` is duplicate, I think it should be `machine-pools`

**Review by vkareh — APPROVED** [review]
/lgtm

File: cmd/ocm/create/machinepool/cmd.go [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+	machinePoolBuilder := cmv1.NewMachinePool()
+
+	machinePoolBuilder = machinePoolBuilder.ID(machinePoolID)
+	machinePoolBuilder = machinePoolBuilder.InstanceType(args.instanceType)
+	machinePoolBuilder = machinePoolBuilder.Replicas(args.replicas)
+
+	if len(labels) > 0 {
+		machinePoolBuilder = machinePoolBuilder.Labels(labels)
+	}
```
Comment (nimrodshn) [inline cmd/ocm/create/machinepool/cmd.go:150]: Nitpick: This build process can be refactored to:
```
machinePoolBuilder.ID(machinePoolID).
    InstanceType(args.instanceType).
    Replicas(args.replicas).
    Labels(labels)
```

As far as I remember its fine for the labels to be an empty.

**Comment by nimrodshn** [comment]
LGTM :100:

**Pull request merged**